src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java

```diff
@@ -0,0 +1,15 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.foundations.tools.GenerateRecordSerializers;
+
+import java.util.HashSet;
+
+public class GenerateRecordSerializersExample {
+    public static void main(String[] args) throws Throwable {
+        GenerateRecordSerializers serializers = new GenerateRecordSerializers();
```

Comment:
Let's move this declaration down to just before it's needed/used.

Reply:
done